maintainers/working_groups/learning_journey/documentation-structure.md

```diff
@@ -0,0 +1,24 @@
+- Tutorials
+    - Nix
+    - NixOS
+- How-To
+    - Language guides
+    - Workflows
+        - Nix REPL
+        - IDE integration
+        - CI
+        - Operations
+        - Containers
+        - Shells
+    - Package management
+    - Contributing
+- Explanation
+    - Nix store
+    - Cross-compilation
+    - Packaging conventions
+- Reference
+    - Nix language reference
+    - Commands reference
+    - Derivations
+    - Nix built-ins
+    - Nixpkgs lib
```

Comment:
These should just be links to the reference manuals. They are self-contained and should be under curation of the code maintainers. That's how we do it right now and there is little reason (or possibility or capacity, in fact) to change that.